tests/rpmbuild.at

```diff
@@ -131,6 +131,37 @@ runroot rpmbuild -bs /data/SPECS/specstep.spec 2>&1|grep ^Executing|cut -d: -f1
 [])
 AT_CLEANUP
 
+# ------------------------------
+# Check if rpmbuild -ba *.spec works after 2038
+AT_SETUP([rpmbuild -ba *.spec after 2038])
+AT_KEYWORDS([build])
+RPMDB_INIT
+AT_CHECK([[
+touch -t 204205042355 ${RPMTEST}/data/SPECS/hello.spec
+touch -t 204205042355 ${RPMTEST}/data/SOURCES/hello-1.0.tar.gz
+touch -t 204205042355 ${RPMTEST}/data/SOURCES/hello-1.0-modernize.patch
+
+unset SOURCE_DATE_EPOCH
```

Comment:
Rather than unset, why not take advantage of it? Ie, point SOURCE_DATE_EPOCH into future?

Reply:
There are also various other tests you can do with that, use SOURCE_DATE_EPOCH as the package buildtime, which will allow testing day/date header format extensions.